README.md

```diff
@@ -9,7 +9,9 @@ It also contains the CLAiR, the [Command Line Applications in Rust][clair] book.
 
 - [Working groups?](https://internals.rust-lang.org/t/announcing-the-2018-domain-working-groups/6737)
 - [Announcement of this WG](https://internals.rust-lang.org/t/announcing-the-cli-working-group/6872/1)
-- [Chat with us](https://gitter.im/rust-lang/WG-CLI)
+- Chat with us
+  - [Discord](https://discord.gg/dwq4Zme)
+  - [Gitter](https://gitter.im/rust-lang/WG-CLI)
```

Comment:
i think we should remove gitter from the list because even though the channel still exists, we can't commit to answering questions there in future

Reply:
I can push another commit to this branch doing that, anyone else have any thoughts?

Reply:
@killercup thoughts?

Reply:
We could always move to the [zulip](http://rust-lang.zulipchat.com/)

:sweat_smile: *hides*